src/Cache.php

```diff
@@ -130,6 +130,9 @@ private function getVersions(): array
         $this->downloader = null;
         $okVersions = [];
 
+        if (!isset($versions['splits'])) {
+            throw new \LogicException('It appears that the flex://defaults entry is missing in the extra.symfony.endpoint array of your composer.json.');
```

Comment:
The message should tell that the endpoint is missing a "splits", which is required to make the `extra.symfony.require` config option work.

Reply:
I'm not sure I understand why you're saying that.

As per [#836](https://github.com/symfony/flex/issues/836), the error occurs when the extra.symfony.endpoint array in composer.json does not include flex://defaults. My ``composer.json`` has never had a ``splits``.

If I have a ``flex://defaults`` array entry, then the ``$versions['splits']`` entry is set. Remove the ``flex://defaults`` array entry and ``$versions['splits']`` is not set.

Reply:
If you want to give me the exact wording of the exception, then I will change the message accordingly. 

Unfortunately I cannot formulate the message when I don't understand what you're referring to.

Reply:
Yeah sure, I was surprised to read your sentence also, but now I get why you phrased it this way. Here is my proposal:

`The Flex index is missing a "splits" entry. Did you forget to add "flex://defaults" in the "extra.symfony.endpoint" array of your composer.json?`

Reply:
Thank you! I rephrased the excepetion message as recommended.